scripts/swarm-deploy.sh

```diff
@@ -69,9 +72,9 @@ do
     # copy config
     docker cp ${IROHA_HOME}/iroha.conf ${COMPOSE_PROJECT_NAME}_node_1:/
     # generate genesis block based on list of peers
-    docker exec ${COMPOSE_PROJECT_NAME}_node_1 /opt/iroha/iroha-cli --genesis_block --peers_address /peers.list
+    docker exec ${COMPOSE_PROJECT_NAME}_node_1 $cli --genesis_block --peers_address /peers.list
```

Comment:
did you do that with @lebdron? or just simply rebased?

Reply:
I changed it.